spartan/bootstrap.sh

```diff
@@ -137,6 +138,12 @@ case "$cmd" in
   "test-prod-deployment")
     FRESH_INSTALL=false INSTALL_METRICS=false ./scripts/test_prod_deployment.sh
     ;;
+  "test-cli-upgrade-with-lock")
+    env
+    FRESH_INSTALL=false INSTALL_METRICS=false \
```

Comment:
INSTALL_METRICS=false however 1-validators.yaml runs with telemetry enabled, might want to override that to not pollute log output

Reply:
also with 1-validators.yaml this test will use full slot / block times and will take longer than it needs to